plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -70,12 +71,18 @@ func (p *PgBouncer) Gather(acc telegraf.Accumulator) error {
 		for col, val := range columnMap {
 			_, ignore := ignoredColumns[col]
 			if !ignore {
-				fields[col] = *val
+				// these values are returned by pgbouncer as strings, which we need to convert.
+				fields[col], _ = strconv.ParseUint((*val).(string), 10, 64)
```

Comment:
This will cause a panic with older pgbouncer versions, since then it came in as an int64, easy fix though I'll just take care of it post merge.

Reply:
cb2c26a3324964d432bf83f535eca7b4d7879db3